lindera/README.md

```diff
@@ -29,14 +29,14 @@ It will:
 
 ```rust
 use lindera::tokenizer::Tokenizer;
-use lindera_core::core::viterbi::Mode;
+use lindera_core::LinderaResult;
 
-fn main() -> std::io::Result<()> {
+fn main() -> LinderaResult<()> {
     // create tokenizer
-    let mut tokenizer = Tokenizer::new(Mode::Normal, "");
+    let mut tokenizer = Tokenizer::new()?;
```

Comment:
Should we explain the default of Mode?

Reply:
We'll document it here next time.

https://github.com/lindera-morphology/lindera/blob/f5b3abb0180f3f4b95e1336ed5ddd347619a9863/lindera/src/tokenizer.rs#L53-L57

https://github.com/lindera-morphology/lindera/blob/f5b3abb0180f3f4b95e1336ed5ddd347619a9863/lindera/src/tokenizer.rs#L22-L36